service/src/main/java/bio/terra/catalog/api/PublicApiController.java

```diff
@@ -44,4 +64,10 @@ public ResponseEntity<VersionProperties> getVersion() {
   public String index() {
     return "redirect:swagger-ui.html";
   }
+
+  @GetMapping(value = "/swagger-ui.html")
+  public String getSwagger(Model model) {
+    model.addAttribute("clientId", swaggerClientId);
+    return "index";
+  }
```

Comment:
SonarQube is complaining about the test coverage for these two methods. It should be relatively easy to unit test these methods.

Reply:
how do I set up a test?